driver/bpf/fillers.h

```diff
@@ -2239,6 +2251,33 @@ static __always_inline bool get_exe_writable(struct task_struct *task)
 	return false;
 }
 
+static __always_inline bool get_exe_upper_layer(struct inode *inode)
+{
+	struct super_block *sb = _READ(inode->i_sb);
+	if(sb == NULL)
+	{
+		return false;
+	}
+
+	unsigned long sb_magic = _READ(sb->s_magic);
+	if(sb_magic == PPM_OVERLAYFS_SUPER_MAGIC)
+	{
+		struct dentry *upper_dentry = NULL;
+		struct inode *lower_inode = NULL;
+		char *vfs_inode = (char *)inode;
+		
+		bpf_probe_read(&upper_dentry, sizeof(upper_dentry), vfs_inode + sizeof(struct inode));
+		bpf_probe_read(&lower_inode, sizeof(lower_inode), vfs_inode + sizeof(struct inode) + sizeof(struct dentry *));
```

Comment:
Pain point here :/ the `struct ovl_inode` structure is already changed:
-> 4.14 https://elixir.bootlin.com/linux/v4.14/source/fs/overlayfs/ovl_entry.h#L60
-> 6.0.9 https://elixir.bootlin.com/linux/latest/source/fs/overlayfs/ovl_entry.h#L127

`struct inode *lower_inode` doesn't exist anymore :/

Reply:
it changes in kernel version `5.19`

Reply:
See previous remark https://github.com/falcosecurity/libs/pull/287#discussion_r973520058

Let's just get rid of the redundant check :) tested in on a 6.0.8 kernel, works:

```
unsigned long sb_magic = _READ(sb->s_magic);
if(sb_magic == PPM_OVERLAYFS_SUPER_MAGIC)
{
	struct dentry *upper_dentry = NULL;
	char *vfs_inode = (char *)inode;
	bpf_probe_read(&upper_dentry, sizeof(upper_dentry), vfs_inode + sizeof(struct inode));
	if(upper_dentry)
	{
		return true;
	}
}
```

Reply:
it would be amazing!!

Reply:
Yes, it seems reasonable! Great Melissa, I think this is the best way to check this.
Updated with this change